nectar/sample-config.toml

```diff
@@ -16,9 +16,11 @@ kraken_api_host = "https://api.kraken.com"
 # The maximum amount of bitcoin to sell in one order, optional field.
 # If absent, orders are capped by the available balance.
 bitcoin = 0.1
-# The maximum amount of dai to sell in one order, optional field.
-# If absent, orders are capped by the available balance.
-dai = 1000
+
+[maker.max_buy]
+# The maximum amount of bitcoin to buy in one order, optional field.
+# If absent, orders are capped by the available quote balance.
```

Comment:
```suggestion
# If absent, orders are capped by the available quote balance given the current price.
```

Reply:
I think it's actually clearer without the suggestion above :)